### PR TITLE
Fix CI and Lint with Prettier

### PR DESCRIPTION
**Pull request by Mahesh209**
This PR fixes the CI so that playwright tests will run. It also addresses our version of codeceptjs which is blocking several security updates from occurring. 
It also adds prettier, updates eslint, updates sitemap, and adds absolute imports.
Prettier was also applied to the template.

### CODE REVIEW

**Comment by Mahesh209** [comment]
@dependabot rebase

File: codecept.conf.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const { setHeadlessWhen, setCommonPlugins } = require('@codeceptjs/configure');
 // turn on headless mode when running with HEADLESS=true environment variable
 // export HEADLESS=true && npx codeceptjs run
-setHeadlessWhen(process.env.HEADLESS);
+setHeadlessWhen(process.env.HEADLESS || true);
```
Comment (Isoji) [inline codecept.conf.js:4]: Will this always evalutate to true?

**Review by Isoji — APPROVED** [review]
Looks good.

File: codecept.conf.js (hunk 1 above)
Comment (Mahesh209) [reply codecept.conf.js]: Yes you're right. I fixed it in the most recent commit.

**Pull request merged**